Assets/BossRoom/Scripts/Shared/Net/GameNetPortal.cs

```diff
@@ -224,6 +227,10 @@ public void StartHost(string ipaddress, int port)
                     unetTransport.ConnectAddress = ipaddress;
                     unetTransport.ServerListenPort = port;
                     break;
+                case UnityTransport UnityTransport:
+                    // UnityTransport. = ipaddress;

```

Comment:
?
Is there anyway to connect to another IP without relay?

Reply:
So you can set the ip and port on the transport

Reply:
We have an IP mode in Boss Room, this commented code should actually be something if we want to use UTP as our default transport, no?

Reply:
I agree with @SamuelBellomo here. We need to still be able to specify the Server address and port. (I can see that they are now private in the adapter) how to support LAN scenarios otherwise? For Relay you need internet.

Reply:
Yes, we will do this change, but in a following PR, as we first need to update the Adapter, then merge that into Netcode GO develop, then upgrade Boss Room and only after that we can use these fields

Reply:
Works for me